src/main/kotlin/no/nav/syfo/behandler/database/domain/PBehandlerArbeidstaker.kt

```diff
@@ -4,18 +4,18 @@ import no.nav.syfo.behandler.domain.*
 import no.nav.syfo.domain.PersonIdentNumber
 import java.time.LocalDateTime
 
-data class PBehandlerDialogmeldingArbeidstaker(
+data class PBehandlerArbeidstaker(
```

Comment:
Hvorfor heter ikke denne `Relasjon`?

Reply:
Var litt i tvil her, men siden databasetabellen heter BEHANDLER_ARBEIDSTAKER så tenkte jeg å ha tilsvarende navn på P-objektet. Men jeg kan godt bruke Relasjon her også hvis det er stemning for det.